redisless/src/command.rs

```diff
@@ -121,6 +122,27 @@ impl Command {
 
                     Error("wrong number of arguments for 'INCR' command")
                 }
+                b"INCRBY" | b"incrby" | b"IncrBy" => {
+                    if v.len() != 3 {
+                        return Error("wrong number of arguments for 'INCRBY' command");
+                    }
+
+                    if let Some(key) = get_bytes_vec(v.get(1)) {
+                        if let Some(increment_bytes) = get_bytes_vec(v.get(2)) {
+                            if let Ok(increment_str) = std::str::from_utf8(&increment_bytes[..]) {
+                                if let Ok(increment) = increment_str.parse::<i64>() {
```

Comment:
Does `INCRBY` should accept **i64** instead of **u64**? Is it allowed to do `INCRBY key -10`?  If yes, can you add a test for that case? Ty

Reply:
Yes, I double check this one because sound weird for me too but

https://redis.io/commands/incrby

```
64 bit signed integer
```

I tested and works as expected 

```
127.0.0.1:6379> SET mykey "10"
OK
127.0.0.1:6379> INCRBY mykey 5
(integer) 15
127.0.0.1:6379> INCRBY mykey -5
(integer) 10
```

And also there is a test :)